### DISCUSSION TITLE
Cell type identification without scRNA-Seq data.

### DISCUSSION THREAD

**Discussion by genomicworks** [GreenleafLab/ArchR#1916] [Questions / Documentation]
Hi, everybody. 

I'm continuously try to set-up and understand the data analysis workflow using ArchR. 
During following "full tutorial", I got question. 

After calculating gene scores, 
how can I identify the cell type in each cluster? (without scRNA-Seq data)
Full tutorial looks not show the method how to. Just suggested to combine with scRNA-Seq data. 

Then, if cell type identification is needed after scATAC-Seq, always scRNA-Seq have to be performed simultaneously?
or I did not catch the methods from tutorial?

I appreciate every answers and comments.
Thank you very much firstly.

**Comment by rcorces** [comment]
If you dont have RNA-seq, then just use gene scores for marker genes of your cell types

**Reply by genomicworks** [reply]
Thank you for answer, corces. 

Is there any vinegettes, tutorials, protocols to perform it?
Just "use gene score" really embarrassed me. 

If you can, please show me the way. 
Thank you firstly.
